src/Parsers/ASTAlterQuery.cpp

```diff
@@ -75,6 +83,53 @@ ASTPtr ASTAlterCommand::clone() const
     return res;
 }
 
+String ASTAlterCommand::typeToString(ASTAlterCommand::Type type)
```

Comment:
Consider to use const char * as return type.

Reply:
Done.